src/cryptography/x509/oid.py

```diff
@@ -151,6 +157,17 @@ class CertificatePoliciesOID(object):
     NameOID.PSEUDONYM: "pseudonym",
     NameOID.DOMAIN_COMPONENT: "domainComponent",
     NameOID.EMAIL_ADDRESS: "emailAddress",
+    NameOID.JURISDICTION_COUNTRY_NAME: (
+        "jurisdictionOfIncorporationCountryName"
```

Comment:
Per https://cabforum.org/2014/03/24/ballot-119-remove-ofincorporation-from-oid-descriptions-in-evg-9-2-5/ these OIDs no longer have `OfIncorporation` in their short names. (Kind of a pedantic point given the difficulty in having canonical names for OIDs I guess)